rune/libenclave/internal/runtime/pal/skeleton/Makefile

```diff
@@ -9,16 +9,13 @@ ENCL_CFLAGS := -Wall -Werror -static -nostdlib -nostartfiles -fPIC \
 	       -fno-stack-protector -mrdrnd -std=gnu11
 HOST_LDFLAGS := -fPIC -shared -Wl,-Bsymbolic
 
-IS_OOT_DRIVER := $(shell [ ! -e /dev/isgx ])
-IS_SGX_FLC := $(shell lscpu | grep -q sgx_lc)
+IS_SGX_FLC ?= $(shell if lscpu | grep -q sgx_lc; then echo "1"; else echo "0"; fi;)
 
 PRODUCT_ENCLAVE ?=
 
 TEST_CUSTOM_PROGS := $(OUTPUT)/encl.bin $(OUTPUT)/encl.ss $(OUTPUT)/liberpal-skeleton-v1.so $(OUTPUT)/liberpal-skeleton-v2.so $(OUTPUT)/liberpal-skeleton-v3.so $(OUTPUT)/signing_key.pem
 
-ifeq ($(IS_OOT_DRIVER),1)
-    TEST_CUSTOM_PROGS += $(OUTPUT)/encl.token
-else ifeq ($(IS_SGX_FLC),)
+ifeq ($(IS_SGX_FLC),0)
     TEST_CUSTOM_PROGS += $(OUTPUT)/encl.token
 endif
 
```

Comment:
it's better to add  as below in Makefile
ifeq ($(IS_SGX_FLC),0)
$(OUTPUT)/encl.token: $(OUTPUT)/encl.ss
        sgx-tools gen-token --signature encl.ss --token $@
endif

ifeq ($(IS_SGX_FLC),0)
EXTRA_CLEAN += $(OUTPUT)/encl.token
endif